includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -647,6 +646,52 @@ public function stats_boxes() {
 		return apply_filters( 'sensei_analysis_stats_boxes', $stats_to_render );
 	}
 
+	/**
+	 * Get the last user activity date.
+	 * It is based on the date on which the last lesson was submitted (completed).
+	 *
+	 * @since 4.2.0
+	 *
+	 * @param  int $user_id
+	 * @return string
+	 */
+	private function get_last_activity( int $user_id ): string {
+		$last_activity_comment = Sensei_Utils::sensei_check_for_activity(
+			array(
+				'number'  => 1,
+				'user_id' => $user_id,
+				'type'    => 'sensei_lesson_status',
+				'status'  => [ 'complete', 'passed', 'ungraded', 'failed' ],
+			),
+			true
+		);
+
+		if ( ! $last_activity_comment ) {
+			return '';
```

Comment:
I believe we wanted to display "N/A" in this case.

Reply:
Updated in d0575dbd38c46d5cd3d01d9ee29e22bdca1756ff.